### PR TITLE
Change navbar colour to green

### PR DESCRIPTION
**Pull request by DanielGordon1**
The PM wanted this change ASAP.

@BenLindner could you look at this please!?

Cheers 👍

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -41,7 +41,7 @@
   font-size: 14px;
 }
 .navbar-wagon-link:hover {
-  color: #da552f;
+  color: green;
```
Comment (DanielGordon1) [inline app/assets/stylesheets/components/_navbar.scss]: This is an epic colour.

**Comment by BenLindner** [comment]
such good much Wow

File: app/assets/stylesheets/components/_navbar.scss (hunk 1 above)
Comment (DanielGordon1) [reply app/assets/stylesheets/components/_navbar.scss]: What do you think @BenLindner

Comment (BenLindner) [reply app/assets/stylesheets/components/_navbar.scss]: pink would be better...

**Pull request merged**